tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1083,6 +1083,7 @@ public function theUserCreatesAFolderUsingTheGraphApi(
 		string $folder,
 		string $spaceName
 	): void {
+		$folder = \trim($folder, "/");
```

Comment:
```suggestion
		$folder = \trim($folder, '/');
```
Use of double `"` and single `'` quotes. Please make it consistent.